CMakeLists.txt

```diff
@@ -689,6 +690,12 @@ ENDIF(MSVC AND PCRE2_STATIC_RUNTIME)
 
 ADD_DEFINITIONS(-DHAVE_CONFIG_H)
 
+if(PCRE2_DEBUG STREQUAL "IfDebugBuild")
+  ADD_COMPILE_DEFINITIONS("$<$<CONFIG:Debug>:PCRE2_DEBUG>")
```

Comment:
requires cmake 3.12, so need to update required version

Reply:
Thanks, done